packages/opentelemetry-instrumentation-openai/opentelemetry/instrumentation/openai/shared/chat_wrappers.py

```diff
@@ -777,11 +777,16 @@ def _accumulate_stream_items(item, complete_response):
         complete_choice = complete_response.get("choices")[index]
         if choice.get("finish_reason"):
             complete_choice["finish_reason"] = choice.get("finish_reason")
+        if choice.get("content_filter_results"):
```

Comment:
this is only part of the solution - as it will only work for streaming results. You need to update this in more places - see for example `_set_completions`. 

Can you also verify me that this is how this field is returned in streaming? Can you link to the docs from Azure?

Reply:
I see. I'll go through the rest of chat_wrappers and let you know if I have other questions.

Here is an example of how the filed is returned in streaming [link](https://learn.microsoft.com/en-us/azure/ai-services/openai/concepts/content-filter?tabs=warning%2Cuser-prompt%2Cpython-new#sample-response-stream-blocked-by-filters). I think the field is _content_filter_results_. Also it seems that _content_filter_results_ won't be returned for non-streaming prompts [link](https://platform.openai.com/docs/guides/chat-completions/response-format) but there is no harm to preempt that.

I do notice that it might be  AzureAI specific. In open AI docs, _content_filter_ is one of the finish reasons but there is no mentioning of _content_filter_results [link](https://platform.openai.com/docs/guides/chat-completions/response-format). I don't think it's gonna cause a problem though

Reply:
I think this is the example for non-streaming:
https://learn.microsoft.com/en-us/azure/ai-services/openai/concepts/content-filter?tabs=warning%2Cuser-prompt%2Cpython-new#example-scenario-an-input-prompt-containing-content-that-is-classified-at-a-filtered-category-and-severity-level-is-sent-to-the-completions-api

Reply:
You're right 😅 I pasted the wrong link. Sounds like we want to handle filtered prompts as well. I think I have an idea how to take care of that as well..